hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/util/BackupUtils.java

```diff
@@ -762,4 +773,30 @@ public static String findMostRecentBackupId(String[] backupIds) {
     return BackupRestoreConstants.BACKUPID_PREFIX + recentTimestamp;
   }
 
+  public static void rollWALWriters(Admin admin, Map<String, String> props) throws IOException {
+    byte[] ret =
```

Comment:
We do not want to introduce a new admin method for this?

Reply:
a. it is not general enough for `Admin`. This call will not only make all rs roll WAL writers, but also do some backup-related operations, such as reading and writing BackupSystemTable.

b. this operation is a bit too lightweight if introduced in the `BackupAdmin`, since it's only a small subprocedure of the whole backup job. 

So I think maybe a static utility method is enough ?

Reply:
The execProcedure call is for zk based procedures, do we still have other procedures besides the log roll one?

Reply:
No, this is the last one.